src/app.js

```diff
@@ -27,3 +27,7 @@ ReactDOM.render(
   </Provider>,
   document.getElementById('app')
 );
+
+window.Intercom("boot", {
+  app_id: "scseqm3j"
```

Comment:
+1. I've created a ticket to track this: https://jira.thedataloom.com/browse/LOOMWEB-173. Leaving it out is good enough for MVP, but we definitely want to track this ASAP